judgels-backends/judgels-grader-app/Dockerfile

```diff
@@ -40,6 +42,9 @@ RUN apt-get update \
     && ln -s /usr/bin/gcc-11 /usr/bin/gcc \
     && ln -s /usr/bin/g++-11 /usr/bin/g++ \
     && curl -sSf https://sh.rustup.rs/ | bash -s -- -y -q --default-toolchain=1.57.0 \
+    && curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.5/install.sh | bash \
```

Comment:
I think it is safer to install Node from the "official" Ubuntu way, not via third-party nvm

https://www.linode.com/docs/guides/install-nodejs-on-ubuntu-22-04/

What do you think?

Reply:
Yes, I will change it. However, the setup that the link you provided, according to https://github.com/nodesource/distributions, the installation scripts 'setup_XX.x' are no longer supported. So, I will follow the official guide instead(adding keyrings, etc)